### ISSUE TITLE
germline module merge multiple samples

### ISSUE THREAD

**Issue by tkcaccia** [KChen-lab/Monopogen#95]
While the preprocessing module works perfectly on multiple samples generating different Bam files for each samples.

The germline module will overwrite the file without considering separaing the files for different samples

**Comment by jinzhuangdou** [comment]
Yes, this is the current design for Monopogen. You need to specific different working folder when running single sample or multip-sample modes.
